content/markdown/02-working-with-data/02-07-dates.Rmd

```diff
@@ -0,0 +1,108 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Working with Dates            
+**Author:** Cole Brookson
+**Date:** 20 July 2022
+
+```{r, echo=FALSE}
+options(tidyverse.quiet = TRUE)
+```
+
+Along with factors, dates are one of the other datatypes that can be a nuisance to work with. EEB-type samples are often taken at inconsistent sampling intervals, and we don't always keep this information in concise formats. 
```

Comment:
I would add something to the first sentence, again to just soften the edges a tiny bit. What about "Along with factors, dates are one of the other datatypes that can be a nuisance to work with, though they are also often essential for understanding our data." Something like that?